### PR TITLE
ui: fix Overview screen in OSS builds

### PR DESCRIPTION
**Pull request by davepacheco**
Previously, when using OSS builds (created with `make buildoss`), when
you loading the DB Console in your browser, you'd get "Page Not Found".
The route for the overview page was missing the leading '/'.  This bug
appears to have been introduced in
722c93252eb0199f23b395cabef1bd416534a441.

Release note (admin ui change): This fixes a bug where users of
the OSS builds of CockroachDB would see "Page Not Found" when loading
the Console.

### CODE REVIEW

**Comment by davepacheco** [comment]
Debugging and fix by @jclulow.  All I did was find the bug and package up the fix that we found to work.  This passes `make ui-lint` and `make buildoss` produces a build where the Console screen works instead of saying "Page Not Found".

**Comment by davepacheco** [comment]
I have signed the CLA several minutes ago but it still says pending, even after I click "recheck".

**Comment by dhartunian** [comment]
Hi @davepacheco and @jclulow! Thank you for your contribution. Let me look into the CLA issue and see what's going on.

**Comment by dhartunian** [comment]
@davepacheco can you try the fix described here https://github.com/cockroachdb/cockroach/pull/55202#pullrequestreview-506215941 and see if that helps? Apologies for the trouble.

**Review by dhartunian — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/56591#-:-MLx8TGC0ptdLLnU33Of:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @dhartunian)

**Comment by dhartunian** [comment]
I'll merge this via bors once CLA is green.

**Comment by davepacheco** [comment]
Thanks @dhartunian.  I tried both fixes in the comment that you linked to.  The first failed like this:

```
$ curl -H "Authorization: token $token" -d '{"state": "success", "context":"license/cla", "description": "curl"}' https://api.github.com/repos/cockroachdb/cockroach/statuses/cc1c94eb5119dad49935ba2e0121c2a17fc8eab7
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#create-a-commit-status"
}
```

That documentation URL implied you can only do this with push access to the repo, so I gave up on that.

The other suggestion was squash and force push.  I just did that, though it doesn't look like the CLA status has been updated.  Let me know if there's something else for me to try.

**Comment by dhartunian** [comment]
appreciate your patience @davepacheco. I confirmed that the CLA is signed and we did the override for the github check.

**Comment by davepacheco** [comment]
Thanks!

**Pull request merged**
